### DISCUSSION TITLE
Remy Sky mod idea

### DISCUSSION THREAD

**Discussion by Rem2x** [thatskymod/Sky-CotL-Scripts#47] [Ideas]
It would be nice to be able to have unlimited white candles

**Comment by thatskymod** [comment]
Having spent a significant portion of my career as a software developer and delving deep into various systems and APIs, I have a comprehensive understanding of how online platforms operate. I appreciate your idea, but how exactly do you envision this working, especially considering that Sky is an online game?

### Let's consider the following experiment:
First, perform a fuzzy memory search to identify in-game currencies, a technique commonly used by scripts and mods. Modify these values to any amount of your choosing, spend some of these virtual candles, and then restart the game to observe the results.

You'll notice that the candle count is reset to its genuine, original state. This occurs because your local edits do not commit as actual transactions to Sky's centralized database. Think of it as faking your bank statement: you can pretend to have millions in your account, but without an official transaction or approval, your actual balance remains unchanged.

However, much like every other online service, Sky exposes an API to its end users. This API acts as a controlled lever to the database. For example, you might tell the server, "Hey, I've collected candles with IDs 507 and 290." The server then verifies and processes your claim. This is precisely what AutoWax does: it convinces the server that you've collected each known candle across the realms. But do note that there's a cap on the number of candles available daily, which is about 21 candles per 24 hours.

**Comment by Lychtinet** [comment]
https://iosgods.com/topic/171061-sky-children-of-the-light-cheats-v0211-84/
Hi, please check this thread out. A modder at iOSGods managed to make these cheese back in the day. I have no knowledge in coding whatsoever but if these can be recreated it would be nice.

**Comment by thatskymod** [comment]
This question appears to have been answered.

Due to prolonged inactivity on this topic, it seems appropriate to close.

If you have any further questions, feel free to reach out!
